multicluster/controllers/multicluster/member/gateway_controller.go

```diff
@@ -73,6 +84,11 @@ func NewGatewayReconciler(
 //+kubebuilder:rbac:groups=multicluster.crd.antrea.io,resources=clusterinfoimports/finalizers,verbs=update
 
 func (r *GatewayReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	if !r.checkClusterSetReady() {
```

Comment:
Could not we just depend on `GetRemoteCommonAreaAndLocalID` to know the ClusterSet is ready?

Reply:
Yeah, `GetRemoteCommonAreaAndLocalID` should replace this since we only map ClusterSet event to corersponding objects when the ClusterSet status is updated to ready. `GetRemoteCommonAreaAndLocalID` will definitely return a common area when ClusterSet status is true. I will refine this part.

Reply:
Done

Reply:
`GetRemoteCommonAreaAndLocalID` can return different errors for different cases. Could you check if they should be treated the same way or not? (If not, I actually wonder why the func should return multiple errors).

Reply:
`GetRemoteCommonAreaAndLocalID` return two kinds of errors, one is that the common area is not initialized at all and not common area, another case is the common area is disconnected and it will return error and nil common area.
In reconcile perspective, we can treat them the same way since there is unnecessary to do object reconcile if there is no common area or no connection to any leader.